maintainers/maintainer-list.nix

```diff
@@ -378,6 +378,12 @@
     githubId = 2822871;
     name = "Alistair Bill";
   };
+  alirezameskin = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add alirezameskin``.